### ISSUE TITLE
Samba mount slow transfer speed vs mounting at command line

### ISSUE THREAD

**Issue by 2xffwd** [linuxmint/nemo#1304]
Hello,

I am using Mint 18

I do not know if this is a limitation of Nemo or it is fixable, but it kinda degrade the experience.

If you browse your samba shares at Nemo and then transfer a file you get half the speed versus mounting the share first at the command line with:

sudo mount -t cifs //192.168.1.10/share /mnt/share -o username=jon,password=password

And then browse the mounted share at Nemo and make the file transfer....for my setup it goes from 45MB/s to 100MB/s

I was getting the full speed doing the same transfer at Windows and i tried multiples configs like tweaking smb.conf to try to get the same speed but it was Nemo...

I found a post from a while back that has the same issue but no solution:

https://forums.linuxmint.com/viewtopic.php?t=205332 (reply 4)

**Comment by leigh123linux** [comment]
It  isn't a limitation of Nemo.
The slow speed is a gvfs-smb problem.

**Comment by inf3rno** [comment]
There is a years old bug report here https://bugzilla.samba.org/show_bug.cgi?id=10879

**Comment by danger89** [comment]
I think this issue is related (and exists already from the year 2013):
https://github.com/linuxmint/nemo/issues/424

**Comment by inf3rno** [comment]
@leigh123linux Isn't it possible to use something different than gvfs-smb if it apparently does not work properly and not fixed for 5+ years?

**Comment by leigh123linux** [comment]
@inf3rno  Have you tried cifs mounts?

https://access.redhat.com/solutions/448263

**Comment by inf3rno** [comment]
@leigh123linux Ofc. I tried, but that is not the default and there was no password popup for that, so either I hardcode the password for the drive into a text file or I don't use it...

**Comment by jeroenst** [comment]
This bug is also in Mint 19.2

**Comment by jeroenst** [comment]
https://bugs.launchpad.net/ubuntu/+source/gvfs/+bug/1236619

**Comment by inf3rno** [comment]
@jeroenst Yes, we have reported this bug in the last 5 years on many forums, but apparently nobody wants to fix it:
https://bugzilla.gnome.org/show_bug.cgi?id=738535
https://bugzilla.gnome.org/show_bug.cgi?id=762384
https://bugzilla.samba.org/show_bug.cgi?id=10879
